ecosystem/sep-0006.md

```diff
@@ -1194,10 +1154,16 @@ All assets listed in a `deposit` and `deposit-exchange` can contain these attrib
 - `authentication_required`: Optional. `true` if client must be [authenticated](#authentication) before accessing the
   deposit endpoint for this asset. `false` if not specified.
 - `fields` (**Deprecated**, Accepting personally identifiable information through request parameters is a security risk
-  due to web server request logging. KYC information should be supplied to the Anchor via SEP-12) `fields` object is
+  due to web server request logging. KYC information should be supplied to the Anchor via SEP-12. For extra fields `types` should be used) `fields` object is
   explained below.
+- `types`: a field with each type of deposit supported for that asset as a key. Each type can specify a `fields`
+  object as below explaining what fields are needed and what they do. Anchors can define custom fields if
+  necessary. If a `types` object is not specified, the wallet should assume that no extra fields are needed for that
+  type of deposit. In the case that the Anchor requires additional fields for a deposit, it should set the
+  transaction status to `pending_customer_info_update`. The wallet can query the `/transaction` endpoint to get the
+  fields needed to complete the transaction in `required_customer_info_updates` and then use
```

Comment:
This stops mid-sentence. I also think you mean `pending_transaction_info_update` and `required_info_updates`.